modules/fedimint-dummy/src/lib.rs

```diff
@@ -74,15 +79,25 @@ impl ModuleGen for DummyConfigGenerator {
         Ok(Dummy::new(cfg.to_typed()?).into())
     }
 
+    fn get_database_migration_data(&self) -> DatabaseMigrationData {
+        let mut migrations = MigrationMap::new();
+
+        migrations.insert(DatabaseVersion(0), move |dbtx| {
+            migrate_dummy_db_version_0(dbtx).boxed()
+        });
+
+        DatabaseMigrationData {
+            kind: Self::KIND,
+            target_db_version: Self::DATABASE_VERSION,
```

Comment:
Both of `kind` and `target_db_version` are statics. I think we should just have a standalone methods to return them and avoid the vague "*Data" aggregate struct.

Reply:
Removed `DatabaseMigrationData` struct